game/sample_game/nakama/private_key.go

```diff
@@ -0,0 +1,163 @@
+package main
+
+// private_key.go manages the creation and loading of the Nakama private key used to sign
+// all transactions.
+
+import (
+	"context"
+	"crypto/ecdsa"
+	"encoding/hex"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strconv"
+	"sync"
+
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/heroiclabs/nakama-common/runtime"
+)
+
+var (
+	ErrorNoStorageObjectFound       = errors.New("no storage object found")
+	ErrorTooManyStorageObjectsFound = errors.New("too many storage objects found")
+
+	globalPrivateKey    *ecdsa.PrivateKey
```

Comment:
i think there are security implications that come with storing pk in memory, we may want to open a ticket to address.

Reply:
Added ticket: https://linear.app/arguslabs/issue/NAK-5/review-the-pattern-of-storing-the-nakama-private-key-in-a-global